.github/workflows/ci_e2e_workflow.yml

```diff
@@ -64,7 +64,7 @@ jobs:
           python -m pip install --upgrade pip
```

Comment:
keep a ToDo task in your notes or ADO task to remove this file and pipeline once ADO pipelines are successfully working and added to repo rulesets

Reply:
Added task in planning repo